Dockerfile

```
@@ -70,14 +72,9 @@ RUN if [ -f /cudf/docker/package_versions.sh ]; \
 ENV CC=/usr/bin/gcc-${CC}
 ENV CXX=/usr/bin/g++-${CXX}
 RUN source activate cudf && \
-    mkdir -p /cudf/cpp/build && \
-    cd /cudf/cpp/build && \
-    cmake .. -DCMAKE_INSTALL_PREFIX=${CONDA_PREFIX} && \
-    make -j"$(nproc)" install
+    cd /cudf/ && \
+    ./build.sh libcudf
 
 # cuDF build/install
 RUN source activate cudf && \
-    cd /cudf/python/cudf && \
-    python setup.py build_ext --inplace && \
-    python setup.py install && \
-    python setup.py install
+    cd /cudf/ && ./build.sh cudf
```

Why not build using single command?

```suggestion
    cd /cudf/ && \
    ./build.sh libcudf cudf
```

---

Merged into a single command.